sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -273,10 +273,13 @@ def process_bundle(self, instruction_id):
   def metrics(self):
     return beam_fn_api_pb2.Metrics(
         # TODO(robertwb): Rename to progress?
-        ptransforms=
-        {transform_id:
-         self._fix_output_tags(transform_id, op.progress_metrics())
-         for transform_id, op in self.ops.items()})
+        ptransforms={
+            transform_id:
+            self._fix_output_tags(transform_id, op.progress_metrics())
+            for transform_id, op in self.ops.items()},
+        user=sum(
+            [op.metrics_container.to_runner_api() for op in self.ops.values()],
+            []))
```

Comment:
Nit: Seems like you have an extra empty list here?

Reply:
This empty list is the second argument to sum (which defaults to 0, but that doesn't work for lists).